plugins/3MFWriter/ThreeMFWorkspaceWriter.py

```diff
@@ -33,7 +35,7 @@ def setExportModel(self, model: SettingsExportModel) -> None:
         if self._ucp_model != model:
             self._ucp_model = model
 
-    def _write(self, stream, nodes, mode=WorkspaceWriter.OutputMode.BinaryMode):
+    def _write(self, stream, nodes, mode, include_log):
```

Comment:
Include log is a bool trap. I'd also argue that instead of removing the default for the mode, we should just set the include_log to be False by default

Reply:
That's right, I have removed the `mode` default value not because of the `include_log`, but because it is a "private" method and only called at one place, with a value for the `mode`. Restoring the default value and setting `include_log` to `False` is also fine if you prefer.